srm.js

```diff
@@ -110,8 +139,18 @@ class CodeDxApiClient {
         }
     }
 
-    async runAnalysis(projectId, formData) {
-        const response = await this.http.post(`/api/projects/${projectId}/analysis`, formData, { headers: formData.getHeaders() }).catch(rethrowError)
+    async getProjectBranches(projectId) {
+        const branchesResponse = await this.http.get(`/x/projects/${projectId}/branches`).catch(rethrowError)
+        return branchesResponse.data.map(branchData => {
+            return new SrmBranch(branchData.name, branchData.isDefault, branchData.id)
```

Comment:
The `SrmBranch` constructor only accepts two params, so `branchData.id` would be ignored

You can just return `branchesResponse.data` directly since `SrmBranch` is just a direct wrapper around these values

Reply:
Oops yeah, I think I initially kept the `id` param for the `SrmBranch` constructor, but eventually removed it since it wasn't exactly required for the work. And I forgot to remove the reference here.

Reply:
This is updated now anyway - returning `branchesResponse.data` directly.